docs/user/integration/_sidebar.md

```diff
@@ -0,0 +1,5 @@
+- [Telemetry Module](../README.md)
```

Comment:
Please put all navigation in the root sidebar, that will make the webpage much nicer to consume

Reply:
Good idea, much better!

Reply:
But where is the outcome? :P